<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="144607" started-at="2018-10-29T10:03:35Z" finished-at="2018-10-29T10:06:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="144607" started-at="2018-10-29T10:03:35Z" finished-at="2018-10-29T10:06:00Z">
      <class name="mobiletests.SanityTest">
        <test-method status="PASS" signature="initialiseDriver()[pri:0, instance:mobiletests.SanityTest@cb0ed20]" name="initialiseDriver" is-config="true" duration-ms="42213" started-at="2018-10-29T10:03:35Z" finished-at="2018-10-29T10:04:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialiseDriver -->
        <test-method status="PASS" signature="additionTest(java.lang.String, java.lang.String)[pri:1, instance:mobiletests.SanityTest@cb0ed20]" name="additionTest" duration-ms="28229" started-at="2018-10-29T10:04:17Z" data-provider="InputData" finished-at="2018-10-29T10:04:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[20]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[20]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- additionTest -->
        <test-method status="PASS" signature="additionTest(java.lang.String, java.lang.String)[pri:1, instance:mobiletests.SanityTest@cb0ed20]" name="additionTest" duration-ms="17042" started-at="2018-10-29T10:04:46Z" data-provider="InputData" finished-at="2018-10-29T10:05:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- additionTest -->
        <test-method status="PASS" signature="subtractionTest()[pri:2, instance:mobiletests.SanityTest@cb0ed20]" name="subtractionTest" duration-ms="16975" started-at="2018-10-29T10:05:03Z" finished-at="2018-10-29T10:05:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- subtractionTest -->
        <test-method status="FAIL" signature="multiplicationTest()[pri:3, instance:mobiletests.SanityTest@cb0ed20]" name="multiplicationTest" duration-ms="17100" started-at="2018-10-29T10:05:20Z" finished-at="2018-10-29T10:05:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [200]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at mobiletests.SanityTest.multiplicationTest(SanityTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- multiplicationTest -->
        <test-method status="PASS" signature="divisionTest()[pri:4, instance:mobiletests.SanityTest@cb0ed20]" name="divisionTest" duration-ms="16970" started-at="2018-10-29T10:05:37Z" finished-at="2018-10-29T10:05:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- divisionTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:mobiletests.SanityTest@cb0ed20]" name="tearDown" is-config="true" duration-ms="6019" started-at="2018-10-29T10:05:54Z" finished-at="2018-10-29T10:06:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- mobiletests.SanityTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
